IntegrationTests/CommonFiles/RCIntegrationRunner.m

```diff
@@ -26,8 +26,7 @@ - (void)start {
                            appUserID:@"integrationTest"];
 }
 
-- (void)getCustomerInfoWithCompletion:(void (^)(RCCustomerInfo * _Nullable, NSError * _Nullable))completion
-NS_SWIFT_NAME(customerInfo(_:)) {
```

Comment:
this `NS_SWIFT_NAME` was already declared in the header, and having it both there and here causes an error

Reply:
it's probably a copy/pasta issue from when we updated the name of the method from `purchaserInfo`